client/wordpress-com.js

```diff
@@ -108,9 +108,9 @@ const sections = [
 		group: 'sites'
 	},
 	{
-		name: 'settings-seo',
-		paths: [ '/settings/seo' ],
-		module: 'my-sites/site-settings/seo-settings',
+		name: 'settings-traffic',
+		paths: [ '/settings/traffic' ],
+		module: 'my-sites/site-settings/traffic',
```

Comment:
Please verify what happens here with the bundle sizes. Previously the SEO settings were in their own bundle because of how they pulled in the `draft-js` editor (a large amount of code) and I would want to make sure that this is updated to reflect how we are currently trying to optimize the load of Calypso and `/settings`